cp/cp_config.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2019-present Open Networking Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#include "monitor_config.h"
+#include "stdio.h"
+#include "string.h"
+#include "stdlib.h"
+#include "cp.h"
+#include "cp_config.h"
+#include <rte_cfgfile.h>
+
+void 
+config_change_cbk(char *config_file, uint32_t flags)
+{
+	fprintf(stderr, "Received %s . File %s Flags %x \n", __FUNCTION__, config_file, flags);
+	/* Move the updated config to standard path */
+	int ret = system("cp /etc/cp/config/app_config.cfg /opt/cp/config/app_config.cfg");
```

Comment:
why is this hardcoded?

Reply:
couple of other places too. You can have defaults but provide overrides, document them. Also document any new files/directory we create or assume to be present. Do not assume CP to be deployed only using omec helm charts.

Reply:
Right. This callback is received only if /etc/cp/config/app_config.cfg is modified. But yes, if file is not present we just continue. I will add #defines or configuration for this file. One difficult thing as of now is to make sure changes don't disturb other deployments.

Reply:
config change registration does not crash if file path does not exist. 
We have 2 modes of deployment
1. Standalone deployment. ....  
   <I don't have setup...>  But what I understand is..
   ./build/ngic_controlplane -<<< binary
   ../config. <<< one directory up has all the config files available..
2. K8s deployment has following file structure
  /opt/cp/config/*.cfg files {adc_rules.cfg  cp_config.cfg  interface.cfg  meter_profile.cfg  pcc_rules.cfg  sdf_rules.cfg}
  /opt/cp/scripts/spgwc-run.sh


Using relative file path Vs. Using absolute file path. Which one we should do ?
@ajamshed @krsna1729  Your comments pls ? Current proposed code definitely works for K8s deployment.